dataactvalidator/config/rule_documentation.csv

```diff
@@ -47,4 +47,4 @@ C/D2,C11,"Unique PIID, ParentAwardId from file C exists in file D1",Linkage,Cros
 C/D1,C12,"Unique PIID, ParentAwardId from file D1 exists in file C, except D1 records with zero FederalActionObligation",Linkage,Cross file existence,Warning,,"There is PIID, ParentAwardId reported in D1 that links to an award in C, unless there's no new activity in D1.",JW: D1 not included yet?,No,File D1 not yet included
 C/D1,C14,If FAIN is not provided then provide URI ,Element Conditionally Required,Conditional requirement,Warning,,,JW: ready?,Yes,"Including PIID in this rule, one of the three must be present "
 C,C17,Not Required on rows that include values for USSGL accounts balances or subtotals,Element Conditionally Required,Conditional requirement,Warning,,This is for transactions not for USSGL amounts from the financial system.,JW: ready,Yes,Ready
-C,C18,DeobligationsRecoveriesRefundsOfPriorYearByAward_CPE = USSGL(4871+ 4872 + 4971 + 4972). This applies to the award level.,Requirement,Record calculation,Warning,,The first element is a total of the other elements,,Yes,
+C,C18,DeobligationsRecoveriesRefundsOfPriorYearByAward_CPE = USSGL(4871+ 4872 + 4971 + 4972). This applies to the award level.,Requirement,Record calculation,Warning,,The first element is a total of the other elements,,Yes,
```

Comment:
I'm not seeing what the change is here, is there something different between these lines?